src/migrate.ts

```diff
@@ -0,0 +1,27 @@
+import * as S from '@eyedea/syncano'
+
+interface Args {
+  migrationKey: string
+}
+
+class Endpoint extends S.Endpoint {
+  async run({response, users}: S.Core, {args, config}: S.Context<Args>) {
+    if (args.migrationKey === config.CUSTOMERIO_MIGRATION_KEY) {
+      try {
+        const allUsers = await users.list()
+        for (const user of allUsers) {
+          if (user.username && !user.customerId) {
```

Comment:
Syncano users can't have empty username so you can skip first check. I would move `!user.customerId` to `await users.whereNotNull('customerId').list()`

Reply:
whereNull? (I need all users that don't have a customerId, whereNotNull will return a list with users WITH customerId)

Reply:
Right, you need whereNull. My mistake